microservice/src/main/java/com/skytouch/model/ProductEntity.java

```diff
@@ -0,0 +1,91 @@
+package com.skytouch.model;
+
+
+import javax.persistence.*;
```

Comment:
Give format to the code, and avoid wildcards. 
Also why is it a bad practice to use wildcards?